metricbeat/metricbeat.yml

```diff
@@ -9,6 +9,8 @@ metricbeat:
       metricsets: ["info"]
       period: 1s
       hosts: ["127.0.0.1:6379"]
+      network: tcp
```

Comment:
Config changes should always be introduced in beat.yml and then by running `make update` the metricbeat.yml is updated.